### DISCUSSION TITLE
Toggle line wrapping for SFTP files

### DISCUSSION THREAD

**Discussion by janwiesemann** [isontheline/pro.webssh.net#1084] [Ideas]
A option to toggle between line wrappings while editing a file though SFT would be great. I’m not always interested in every detail of a line. Especially long lines take up a huge amount of your screen. I think it should be added into the find/replace menu.

![IMG_6769](https://github.com/isontheline/pro.webssh.net/assets/1347201/cca3ca6a-0723-403f-9089-67092cd8606d)

**Comment by isontheline** [comment]
Thanks @janwiesemann for your feedback 🙏

I'm creating an issue from your idea : https://github.com/isontheline/pro.webssh.net/issues/1086
